### PR TITLE
chore: update docs

### PR DESCRIPTION
**Pull request by jofftiquez**
- add a more "vue way" sample

Thanks for this library <3

### CODE REVIEW

File: examples/vuejs/README.md [hunk 1]
```diff
@@ -2,47 +2,64 @@
...
-```
+### Usage
+
+```vue
+<template>
+  <qrcode-scanner
+    :qrbox="250" 
+    :fps="10" 
+    style="width: 500px;"
+    @decodedQrCode="onScan"
```
Comment (mebjas) [inline examples/vuejs/README.md:52]: I don't see how `@decodedQrCode` is connected. Should the `onScanSuccess` emit `decodedQrCode` as event?

**Review by mebjas — CHANGES_REQUESTED** [review]

**Comment by mebjas** [comment]
Thanks for the change, left one comment!

File: examples/vuejs/README.md (hunk 1 above)
Comment (jofftiquez) [reply examples/vuejs/README.md]: Hi you are right, the emitted event name was `result` so it should be `@result`. I chose the word "result" to make it more explicit and easy to understand. I updated the PR :)

**Review by mebjas — APPROVED** [review]

**Comment by mebjas** [comment]
Thanks for the PR, merging.

**Comment by mebjas** [comment]
Added you as contributor to the doc - https://github.com/mebjas/html5-qrcode/commit/e7f73064e233fa5ef55357af134fbe19dd680a9a

I'll also be adding a global page, thanks!

**Comment by jofftiquez** [comment]
Thanks @mebjas!

**Pull request merged**